.github/workflows/nextjs.yml

```
@@ -0,0 +1,106 @@
+            exit 1
+          fi
+      - name: ⚙️ Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+          cache: ${{ steps.detect-package-manager.outputs.manager }}
+          cache-dependency-path: ${{ env.WORKSPACE_PATH }}/${{ steps.detect-package-manager.outputs.manager == 'yarn' && 'yarn.lock' || 'package-lock.json' }}
+      - name: 🔧 Setup Pages
+        uses: actions/configure-pages@v5
+        with:
+          working-directory: ${{ env.WORKSPACE_PATH }}
```

actionlint (1.7.4)

input "working-directory" is not defined in action "actions/configure-pages@v5". available inputs are "enablement", "generator_config_file", "static_site_generator", "token"

(action)